### DISCUSSION TITLE
Keep Engine Running

### DISCUSSION THREAD

**Discussion by Rabtos** [YimMenu/YimMenu#1210] [Ideas]
Could you add the option "Keep Engine Running" even when I exit the car, please?
So far there are only those 2 options:
[url]https://i.imgur.com/5Xynr8X.png[/url]
I imagine adding something like "Disable Engine Auto Turn Off"
Would love it, cuz you could get the big Party Bus and hear music with other people when chilling outside.

Thank You

**Comment by Tayipc38** [comment]
I suggest you open a request issue instead of a discussion.

**Comment by Yimura** [comment]
Related issue #1290
